app/src/main/java/com/kira/android_base/main/fragments/home/HomeFragment.kt

```diff
@@ -15,7 +32,40 @@ class HomeFragment : BaseFragment(R.layout.fragment_home) {
             mainViewModel = this@HomeFragment.mainViewModel
             lifecycleOwner = viewLifecycleOwner
         }
-
         mainViewModel.getLocalUser()
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val binding = viewDataBinding as FragmentHomeBinding
+
+        binding.listItemHome.apply {
+            adapter = ListPostHomeRecyclerViewAdapter().apply {
+                list.addAll(data.map {
+                    PostItemHomeModel(
+                        it.user,
+                        it.place,
+                        it.time,
+                        it.title
+                    )
+                })
+                listener = object : BaseRecyclerViewAdapter.Listener<PostItemHomeModel> {
+                    override fun onItemClick(position: Int, t: PostItemHomeModel) {
+                        navigateToPostDetail(view)
+                    }
+                }
+            }
+            layoutManager = LinearLayoutManager(
+                context,
+                LinearLayoutManager.VERTICAL,
+                false
+            )
+        }
+    }
+
+    fun navigateToPostDetail(view: View) {
```

Comment:
hàm này nếu chỉ dùng trong class này thì phải thêm private nhé, nguyên tắc là cái nào cần public thì mới public, còn lại private toàn bộ để tránh truy cập linh tinh từ bên ngoài